### PR TITLE
Added GetQueryType() to python query atom API

### PR DESCRIPTION
**Pull request by rachelnwalker**
#### Reference Issue
Addresses #4880

#### What does this implement/fix? Explain your changes.
This is exposes the `getQueryType()` query atom function to python so that atom query types/symbols (Q, QH, X, XH, etc) can be accessed. This is mostly helpful because the query symbols can now be used as a dummy label so that it will render in image generation. Here is an example of what the images will looks before and after setting dummy labels using query atom symbols:
  
before
Screen Shot 2022-01-07 at 4 09 50 PM

#### Any other comments?
This might be something we want to automatically render

### CODE REVIEW

File: Code/GraphMol/Atom.h [hunk 1]
```diff
@@ -259,6 +259,9 @@ class RDKIT_GRAPHMOL_EXPORT Atom : public RDProps {
   // This method can be used to distinguish query atoms from standard atoms:
   virtual bool hasQuery() const { return false; }
 
+  //! NOT CALLABLE
```
Comment (greglandrum) [inline Code/GraphMol/Atom.h:262]: This isn't really accurate.
```suggestion
```

**Review by greglandrum — APPROVED** [review]
One minor suggested doc change, otherwise LGTM

**Comment by rachelnwalker** [comment]
Thanks @greglandrum, I just fixed that

**Review by greglandrum — APPROVED** [review]
LGTM

**Pull request merged**